### DISCUSSION TITLE
Controlling of charge/discharge via UART-TTL works fine, balancer does not

### DISCUSSION THREAD

**Discussion by cflenker** [syssi/esphome-jk-bms#374] [Q&A]
This project is awesome and I am very thankful for that.
I just tried to switch on/off charging and discharging via UART-TTL and it works fine.
Then I tried to include the balancer-switch in the yaml-file, but this did not compile.
Is it a limitation of the jkbms that the balancer can not be switched on/off via UART-TLL but only via bluetooth.
Or is it just not implemented in this project?
Thank you very much!!!

**Comment by syssi** [comment]
Could you provide the error message? It should compile in general and if you don't own a v11 BMS you should be able to enable balancing using this switch (via UART-TTL).

**Reply by cflenker** [reply]
Thanks a lot for the quick response!
I just took **esp32-example.yaml** as a blueprint and added the balancer:

> switch:
>   \- platform: jk_bms
>     charging:
>       name: "${name} charging"
>     discharging:
>       name: "${name} discharging"
>     **balancer:
>       name: "${name} balancer"**

Then I get the following error when trying to compile it:
> [balancer] is an invalid option for [switch.jk_bms]. Please check the indentation.

My BMS is Hardware Version V10.XW and Software Version V10.09.

Thanks a lot!!!

**Reply by syssi** [reply]
Oh, my fault. The balancer switch isn't available yet:

https://github.com/syssi/esphome-jk-bms/blob/main/components/jk_bms/switch/__init__.py#L40-L45

Do you want to give it another try? I will push a change.
